circles/migrations/0002_auto_20150617_2034.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+from django.utils.timezone import utc
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('circles', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='circle',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2015, 6, 17, 18, 33, 58, 802301, tzinfo=utc)),
```

Comment:
These are the times that get written into the database when you migrate. When circles get created/updated, they get the current time.